### ISSUE TITLE
Training script and dataset generate script request.

### ISSUE THREAD

**Issue by fh1999** [serycjon/WOFT#3]
Hi:
    Thanks for open source, I want to reproduce the results in the paper. Could you open source the relevant scripts?

**Comment by serycjon** [comment]
Sorry I have too much work now, probably after ICCV deadline

**Comment by fh1999** [comment]
Hi:
Sorry to have disturbed you. Could you give me some relevant raw script for reference？This will save me a lot of time. Thanks again!

**Comment by serycjon** [comment]
Sorry, cannot share that now, but the loss function is the [torch_reproj_errors](https://github.com/serycjon/WOFT/blob/06e73878be81ab5ac14fc3c6ad62b983d44d9304/pytracking/utils/least_squares_H.py#L400).

**Comment by ailovejinx** [comment]
> Sorry I have too much work now, probably after ICCV deadline

Hi Serych,

I focus on your work WOFT and MFT. I think these are excellent work.

Same as @fh1999 , I also want to replicate your results. I'd appreciate it if you release train and test scripts. 

Thanks!

**Comment by serycjon** [comment]
Hi, I have published the MFT training script and instructions in https://github.com/serycjon/MFT/blob/master/train.org
Sorry about WOFT, I still have it on my todolist.  Will let you know when I publish the training stuff.

**Comment by ailovejinx** [comment]
> Hi, I have published the MFT training script and instructions in https://github.com/serycjon/MFT/blob/master/train.org Sorry about WOFT, I still have it on my todolist. Will let you know when I publish the training stuff.

Thank you and your excellent work again!

**Comment by shank885** [comment]
@serycjon Any update on the training scripts?

**Comment by serycjon** [comment]
Sorry, @shank885 , it has been a long time and I am too swamped now to do the archeology, so to speak.
